browser/ui/webui/brave_webui_source.cc

```diff
@@ -6,8 +6,12 @@
 
 #include <map>
 
+#if !defined(OS_ANDROID)
 #include "brave/components/brave_rewards/resources/grit/brave_rewards_resources.h"
```

Comment:
I thought you were sharing the actual resources now. They are split up so it should be possible